logdna/logdna.py

```diff
@@ -1,72 +1,83 @@
 import sys
 import time
 import json
-from .configs import defaults
 import logging
-import requests
-import threading
 import socket
-from threading import Timer
+import threading
+import requests
+from requests.adapters import HTTPAdapter
+from urllib3.util import Retry
+from .configs import defaults
+from .utils import sanitize_meta
+from .utils import get_ip
+from .utils import get_option
```

Comment:
Format-wise, I'm more used to `from .utils import get_ip, get_option, sanitize_meta`